class UserAlreadyExists(Exception): ...


class AuthFailed(Exception): ...
